mobile-app/lib/features/components/transaction_action_sheet.dart

```diff
@@ -46,6 +46,8 @@ class _ReversibleTransactionActionSheetState extends ConsumerState<ReversibleTra
     return HumanReadableChecksumService().getHumanReadableName(address);
   }
 
+  bool get _isReverseDisabled => _remainingTime == null || _remainingTime == Duration.zero;
+
```

Comment:
is there a legitimate case where _remainingTime == null? That seems like an error?

Reply:
If there's a legit reason, no problem. 

If it's an error, then that variable should be non-nullable instead.